### PR TITLE
Companymodelfeatures

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#56

### CODE REVIEW

**Comment by patmbolger** [comment]
Looks good to me.

File: app/models/company.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Company < ActiveRecord::Base
+  has_many :company_people
+  has_many :jobs
+  has_many :addresses, as: :location
+  has_and_belongs_to_many :agencies
+
+  validates_presence_of :name, :ein, :website, :phone, :email
+  validates_length_of   :name, maximum: 100
+  validates_length_of   :website, maximum: 200
+  validates :ein,   :presence => true
```
Comment (joaopapereira) [inline app/models/company.rb]: This line should be
validates :ein, :ein => true

to run the ein validator

File: spec/models/company_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
```
Comment (joaopapereira) [inline spec/models/company_spec.rb]: Missing EIN tests, something like:

describe 'EIN' do
      it { should_not allow_value('asd', '123456', '123-456789', '12-34567891', '00-0000000', '000000000')
                          .for(:ein)}
      it { should allow_value('12-3456789', '123456789')
                      .for(:ein)}
      it { validate_presence_of :ein}
    end

Comment (joaopapereira) [inline spec/models/company_spec.rb]: We also should have some validations of Email, website and email

**Comment by joaopapereira** [comment]
Please review my previous comments

File: app/models/company.rb (hunk 1 above)
Comment (vijayakarumudi) [inline app/models/company.rb]: validates :ein, :ein => true  generates argument error( Unknown validator Einvalidator)

**Comment by vijayakarumudi** [comment]
I added all the validations.

**Pull request closed without merging**